### PR TITLE
Some packages update

### PR DESCRIPTION
**Pull request by RSKYS**
Hello! It's been a while.
I've been  busy but eventually switched to Slackware for good I guess. Good job to alien that I could test Plasma 6 on ktown preview and it got huge success on wayland so, can't wait.

Notes that I should mention is that some packages like **python-httpx**, **mutagen** and possibly few more have unresolved dependencies in their *.info. I think some dependencies got included in system because I had absolute no problem while building.

I'll update few more packages per my future need again later.

### CODE REVIEW

**Comment by willysr** [comment]
@RSKYS some of the commits were not under your maintenance.
please make a pull request only for those under your maintainership

File: audio/tauonmb/tauonmb.SlackBuild [hunk 1]
```diff
@@ -27,8 +27,8 @@
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=tauonmb
-VERSION=${VERSION:-7.2.1}
-SRCNAM=TauonMusicBox
+VERSION=${VERSION:-7.8.0}
+SRCNAM=Tauon
 BUILD=${BUILD:-2}
```
Comment (willysr) [inline audio/tauonmb/tauonmb.SlackBuild:32]: This should be reset to 1

**Comment by RSKYS** [comment]
> @RSKYS some of the commits were not under your maintenance. please make a pull request only for those under your maintainership

There's only a discord which I bumped it, should I revert the commit?

**Comment by RSKYS** [comment]
> > Notes that I should mention is that some packages like **python-httpx**, **mutagen** and possibly few more have unresolved dependencies in their *.info. I think some dependencies got included in system because I had absolute no problem while building.
> 
> Can you explain further what you mean? And are you talking about on 15.0 or current?

@fourtysixand2  Yes, yes sure

image


It's -current and -15.0 git repositories, some of dependency packages are removed and I'm seeing this warning, although I simply ignore and proceed, it's been fine other than these warns.
See their *.info

File: audio/tauonmb/tauonmb.SlackBuild (hunk 1 above)
Comment (RSKYS) [reply audio/tauonmb/tauonmb.SlackBuild:32]: @willysr I'm gonna fix this later, could you please tell me what to do about [this](https://github.com/SlackBuildsOrg/slackbuilds/pull/6817#issuecomment-2177781763)?

**Comment by willysr** [comment]
> > @RSKYS some of the commits were not under your maintenance. please make a pull request only for those under your maintainership
> 
> @willysr There's only a discord which I bumped it, should I revert the commit? It's version update, and unfortunately discord doesn't work with previous version when there's an available update.

yes, it's @bassmadrigal script, so you should ask him to update discord because he's active maintainer

**Comment by willysr** [comment]
yes, since SBo only focusing on stable releases, you need to test them against -stable, not -current

**Comment by RSKYS** [comment]
> yes, since SBo only focusing on stable releases, you need to test them against -stable, not -current

@willysr Yes, I've built them using both already.

**Comment by RSKYS** [comment]
@willysr Done

**Comment by willysr** [comment]
@sbo-bot: build python/python3-natsort

**Comment by willysr** [comment]
@sbo-bot: build audio/tauonmb

**Comment by willysr** [comment]
@sbo-bot: build python/python3-pulsectl

**Comment by willysr** [comment]
@sbo-bot: build python/python3-plexapi

**Comment by willysr** [comment]
@sbo-bot: build python/python3-pysdl2

**Comment by willysr** [comment]
@RSKYS audio/tauonmb failed to build

**Comment by willysr** [comment]
@sbo-bot: build audio/tauonmb

**Comment by willysr** [comment]
@sbo-bot: build system/kvantum-qt5

**Comment by willysr** [comment]
@sbo-bot: build python/python3-pypresence

**Comment by willysr** [comment]
@RSKYS i will take every commit, except for audio/tauonmb
please send a single commit for that

**Comment by bassmadrigal** [comment]
> > @RSKYS some of the commits were not under your maintenance. please make a pull request only for those under your maintainership
> 
> @willysr There's only a discord which I bumped it, should I revert the commit?

It is SBo policy to only submit updates for packages you don't maintain _after_ contacting the current maintainer **and** getting approval. We've had this dance before, but at least you didn't lie this time (#1016 when you lied about trying to contact me and #1057 when you lied about supposedly contacting me and me saying I no longer use Slackware -- I've used Slackware for over 20 years... I'm not going anywhere).

If you are not the maintainer, you should not submit an update unless you've reached out and received approval.

If the maintainer seems to be keeping things up-to-date (like I've [rarely missed a discord update](https://git.slackbuilds.org/slackbuilds/log/network/discord?h=15.0) in the last 18 months), you should give them the benefit of the doubt that they'll be updating it before pestering them. If it goes a few weeks or they're sporadic in their updates, then it's reasonable to reach out.

If a maintainer appears inactive (months or years since their last update to the repo, not necessarily a single piece of software), you should attempt to reach out to them (and give it a week or two and usually a follow-up reply within that timeframe) before requesting permission from the SBo admins to take over the maintainership of the package.

>It's version update, and unfortunately discord doesn't work with previous version when there's an available update.

This is just what happens with SBo's weekly public release schedule. Any program, even if the maintainer is religious at updating, could go up to a week with a pending update waiting for the public release. I saw this needed to be updated on Monday and was going to ensure an update was submitted before the public update is released.

This is also why there's a message in the README on how to bypass the failing when starting an old version. I did find out today that if one wasn't familiar with json formatting, it would fail as written, so I've included a minor tweak to the README in today's PR (#6825) to hopefully better help others.

You're welcome to maintain your own versions of software on SBo on your own computer, but it is not ok to submit updates for packages you don't maintain without prior approval from the maintainer or SBo admins.

**Comment by RSKYS** [comment]
@bassmadrigal Yes, yes you're right!
For that in past I'm sorry, it was act of unprofessional; but let it stay in the past.
I remember that I could bump some packages of other people, that's why I **only** tried to bump it without major changes.
@willysr told me it's not possible, and I stopped it already.
It's not that I want to disrespect you or anything.

Cheers

**Comment by bassmadrigal** [comment]
> @bassmadrigal Yes, yes you're right For that in past I'm sorry, it was act of unprofessional; but let it stay in the past.

I'll never forget your name after you literally lied to SBo admins twice about contacting me for two different packages. One you lied just saying you couldn't contact me, but the other was far more egregious by saying you did contact me and that I decided to stop using Slackware and no longer maintained my packages.

Absolutely unacceptable!

>And I don't remember why I couldn't reach you!

You never tried either time. I checked my inbox and my email hasn't changed in over a decade.

>it's been 2 years, forgive my sins xD

As you go right back and make new ones by submitting updates for a package you don't maintain without attempting to contact me... again? At least this time didn't include a lie about trying to contact me for permission, but instead you thought it was acceptable to submit an update for a program you don't maintain without permission. 

Both are pretty bad things to do...

>I remember that I could bump some packages of other people, that's why I **only** tried to bump it without major changes.

You remember wrong. This is literally [covered in the FAQ](https://slackbuilds.org/faq/#version_update).

@willysr told me it's not possible, and I stopped it already. 

You were told before and it happened now, but maybe you'll actually learn your lesson this time? Or will I need to wait another two years and have this conversation with you again?

**Comment by RSKYS** [comment]
@bassmadrigal Yes, I've learned my lesson this time and won't miss the track.
Patience is what I didn't have those days, that's why I'm ashamed

**Pull request closed without merging**